### PR TITLE
All notifications

### PR DESCRIPTION
**Pull request by Aiza-anarbekova**

### CODE REVIEW

File: src/main/java/kg/peaksoft/giftlistb6/api/NotificationApi.java [hunk 1]
```diff
@@ -21,15 +18,16 @@ public class NotificationApi {
...
     }
 
-    @GetMapping("/read")
-    public AllNotificationsResponse isRead(){
-        return notificationService.isRead();
+    @Operation(summary = "Mark as read", description = "User can mark as read all notifications")
+    @PostMapping
+    public AllNotificationsResponse isRead() {
+        return notificationService.markAsRead();
     }
```
Comment (nurtilek94) [inline src/main/java/kg/peaksoft/giftlistb6/api/NotificationApi.java]: unnecessary line

File: src/main/java/kg/peaksoft/giftlistb6/db/services/BookedService.java [hunk 2]
```diff
@@ -61,18 +85,27 @@ public SimpleResponse reserveWish(Long wishId, boolean is) {
```
Comment (nurtilek94) [inline src/main/java/kg/peaksoft/giftlistb6/db/services/BookedService.java:85]: unnecessary line

File: src/main/java/kg/peaksoft/giftlistb6/dto/responses/NotificationResponse.java [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+import java.time.LocalDate;
+
+@Getter
+@Setter
+@AllArgsConstructor
+public class NotificationResponse {
+
+    private Long id;
+    private String fullName;
```
Comment (i-Coderr) [inline src/main/java/kg/peaksoft/giftlistb6/dto/responses/NotificationResponse.java]: Remove fullName and add userId, firstName, lastName
Rename createdDate to createdAt

**Comment by i-Coderr** [comment]
Commented at 2.40 pm

**Pull request merged**